Realm/RLMObjectSchema.mm

```diff
@@ -171,6 +171,21 @@ + (instancetype)schemaForObjectClass:(Class)objectClass {
     return schema;
 }
 
++ (bool)_isInvalidLazyProperty:(NSString *)propertyName ignoredProps:(NSArray *)ignoredProperties {
+    NSString *const storageSuffix = @".storage";
+    const NSUInteger suffixLength = [storageSuffix length];
+    if (![ignoredProperties containsObject:propertyName]
+        && [propertyName length] > suffixLength
+        && [propertyName hasSuffix:storageSuffix]) {
```

Comment:
Checking `[propertyName length] > suffixLength` seems redundant since `-hasSuffix:` would have to perform a similar check internally in order to return true.